tsconfig.json

```diff
@@ -29,6 +29,7 @@
     "target": "esnext",
     "allowJs": false,
     "baseUrl": ".",
+    "skipLibCheck": true,
```

Comment:
Needed at the moment for one package. I don't see a major downside. I plan to enable this later again.